src/main/java/de/craftlancer/clfeatures/Feature.java

```diff
@@ -65,7 +64,8 @@ public Feature(CLFeatures plugin, ConfigurationSection config, NamespacedKey lim
             limitConfig.getKeys(false).forEach(a -> limitMap.put(a, limitConfig.getInt(a)));
         
         plugin.getServer().getPluginManager().registerEvents(this, plugin);
-        new LambdaRunnable(() -> getFeatures().forEach(FeatureInstance::tick)).runTaskTimer(getPlugin(), 10, 10);
+        new LambdaRunnable(() -> getFeatures().forEach(FeatureInstance::tick)).runTaskTimer(getPlugin(), 10,
+                this instanceof JukeboxFeature ? 1 : 10);

```

Comment:
Remove instanceof check. An abstract class shouldn't know about who implements it. Add a getTickFrequency() method or so, that can be overwritten by JukeboxFeature